README.md

```diff
@@ -3,8 +3,8 @@
 ## Table Of Content
 
 * [What is the RedHat Che distribution](#what-is-the-redhat-che-distribution)
-* [RedHat Che distribution Maven build](#redhat-che-distribution-maven-build)
 * [Developer-friendly scripts to build and deploy or rollout to minishift](#developer-friendly-scripts-to-build-and-deploy-or-rollout-to-minishift)
+* [Maven build details](#maven-build-details)
 
 ## TL;DR
```

Comment:
This TL;DR doesn't make sense as it is because it just covers the build. You can:
- remove it
- make it a 2 snippets: to build, to deploy. The goal is that people in a hurry can just copy/paste without going through the doc.

I would remove it.